packages/package_info/example/test_driver/package_info_e2e.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
```

Comment:
@collinjackson Was it necessary to create these xxx_e2e.dart files for the new e2e test or we can just update the old xxx.dart to the new e2e test style?

Reply:
CI will only run the tests on Android if they follow the new convention.